tests/components/home_plus_control/test_init.py

```diff
@@ -338,9 +401,18 @@ async def test_module_status_reduction_change(
     )
 
     # Load the entry
-    hass.data[DOMAIN] = {}
-    hass.config.components.add(DOMAIN)
-    await setup.async_setup_component(hass, "http", {})
+    assert await setup.async_setup_component(
+        hass,
+        "home_plus_control",
+        {
+            "home_plus_control": {
+                CONF_CLIENT_ID: CLIENT_ID,
+                CONF_CLIENT_SECRET: CLIENT_SECRET,
+                CONF_SUBSCRIPTION_KEY: SUBSCRIPTION_KEY,
+            },
+            "http": {},
+        },
+    )
     assert hass.http.app
```

Comment:
Remove the http asserts.

Reply:
OK. I cleaned up the tests as per your suggestions. 

Thanks!